installscripts/jazz-terraform-unix-noinstances/scripts/create.sh

```diff
@@ -1,20 +1,41 @@
-#!/bin/bash
+#!/bin/sh
+
+print_error()
+{
+    printf "\r${RED}$1${NC}\n" 1>&3 2>&4
+}
+
 rm -f ./settings.txt
 date
-terraform apply \
-          -var "aws_access_key=${AWS_ACCESS_KEY_ID}" \
-          -var "aws_secret_key=${AWS_SECRET_ACCESS_KEY}" \
-          -var "region=${AWS_DEFAULT_REGION}"
-date
-echo " ======================================================="
-echo " The following stack has been created in AWS"
-echo " ________________________________________________"
-terraform state list
-echo " ======================================================="
-echo " Please use the following values for checking out Jazz"
-echo " ________________________________________________"
-cat ./settings.txt
-echo " ======================================================="
-echo " Installation complete! To cleanup Jazz stack and its resources execute ./destroy.sh in this directory."
-realpath ../../
-echo " ======================================================="
+terraform init && terraform apply \
+                            --auto-approve \
+                            -var "aws_access_key=${AWS_ACCESS_KEY_ID}" \
+                            -var "aws_secret_key=${AWS_SECRET_ACCESS_KEY}" \
+                            -var "region=${AWS_DEFAULT_REGION}"
+if [ $? -gt 0 ]
+then
+    date
+    print_error "$message....Failed"
```

Comment:
Finally we can rollback on failures, super cool!

Reply:
It seems to work OK. It doesn't undo *everything*, but it will destroy all AWS resources Terraform created, which is most of what people actually would want.